### ISSUE TITLE
"TF record not found"

### ISSUE THREAD

**Issue by Rafiur** [ash3n/DocProduct#34]
Upon running the "Train your own medical Q&A retrieval model" colab notebook I found that even though the pandas dataframe can read the file in "data/sampleData.csv" but running the
"d = create_dataset_for_bert(
    'data/sampleData.csv', tokenizer=tokenizer, batch_size=batch_size,
    shuffle_buffer=500000, dynamic_padding=True, max_seq_length=max_seq_len)"
gives an output of "TF record not found"
Also if "d" is printed we can see that all the attributes like question id or question mask etc have none in their value and the end result of running the last cell on the notebook results in a "value error: empty training data"
How can I fix this?

**Comment by Santosh-Gupta** [comment]
@ash3n If you are going to steal this repo from me and @JayYip , could you please at least respond to the issues. We are not motivated to respond to issues on a stolen repo.
